chart/hyrax/min.yaml

```diff
@@ -0,0 +1,19 @@
+ingress:
```

Comment:
Would it be worth referencing the use of this file in the chart's README?

Reply:
oh, we probably should just drop it.

this was for my local testing and got caught up in an overeager git add

Reply:
squashed out the extra `.yaml` files after porting the example to the inline docs.

Reply:
LGTM!